### ISSUE TITLE
Confusability vs string matching

### ISSUE THREAD

**Issue by r12a** [w3c/charmod-norm#87]
http://w3c.github.io/charmod-norm/#normalizationLimitations

"'Confusable' characters, regardless of script, can present spoofing and other security risks. For more information on homoglyphs and confusability, see [UTS39]."

is this strictly relevant to string matching?  Perhaps it should be a note, if it is kept?

**Comment by klensin** [comment]
If the purpose of string matching it to align with user expectations about what will match, then it is certainly relevant.  If it is only to act in a Unicode-consistent way with the expectation that the users will eventually get used to, and accept, what is done, then less so.    In addition, the text (whether in a note or inline) should be careful about a statement that UTS #39 is definitive on this particular issue: not only is the issue closely tied to user perception but, it seems to me that document skirts some of the cases.  In addition, while the removal of WhileScript.txt and other changes in the January 2016 draft version is, IMO, an improvement, it makes the guidance more difficult to understand and apply uniformly.  The new email material is simply wrong, starting with how the structure of email addresses is described.

**Comment by asmusf** [comment]
I find the transition from the previous sentence a bit jarring. So I would reword:

"Regardless of script, any characters that are identical or 'confusable' in appearance  can present spoofing and other security risks. For a discussion and examples of homoglyphs and confusability, see [UTS39]."

For UTS39 perhaps the examples in the data file are at least as interesting as the text. Perhaps phrasing the reference a bit differently, as proposed above, can help sidestep the issue of whether this is considered 'definitive' in this context.

**Comment by aphillips** [comment]
Done as: 		  

> Characters that are identical or 'confusable' in appearance can present spoofing and other security risks. 
>This can be  true within a single script or for similar characters in separate scripts. For a 
> discussion and examples of homoglyphs and confusability, see  [[UTS39]].

Pull request to follow.
